### ISSUE TITLE
Can't be installed with 32 bit?

### ISSUE THREAD

**Issue by durango** [gographics/imagick#19]
```
root@dev:/www/resources/golang# go get github.com/gographics/imagick/imagick
# github.com/gographics/imagick/imagick
/root/gopath/src/github.com/gographics/imagick/imagick/draw_info.go:14:6: unexpected: 12-byte float type - long double
```

**Comment by TatriX** [comment]
I have the same problem on 32 bit system:
```
github.com/gographics/imagick/imagick/draw_info.go:17:19: unexpected: 12-byte float type - long double
```
How could I fix it?

**Comment by justinfx** [comment]
Not sure exactly. I haven't tested a build on a 32 bit system. Patches
welcome if you find the cause.

**Comment by meilihao** [comment]
I have the same problem on 32 bit Fedora22.

ImageMagick-devel       i686      6.8.8.10-9.fc22
go version go1.4.2 linux/386
gcc (GCC) 5.1.1 20150422 (Red Hat 5.1.1-1)

**Comment by justinfx** [comment]
Does this negatively impact windows x64? If it's a general windows fix, I
could make it a Windows build constraint.

**Comment by justinfx** [comment]
Fixed by #59
